src/components/Button/Button.tsx

```diff
@@ -132,6 +133,20 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
       isLoading && styles['button--is-loading'],
     );
 
+    assertEdsUsage(
+      [
+        typeof isDisabled === 'undefined' &&
+          typeof other.disabled !== 'undefined',
+      ],
+      'Use "isDisabled" instead of "disabled" on button instances',
```

Comment:
This would be when using `disabled` instead of `isDisabled`, that situation where the UI shows  that red dotted outline around a button (e.g., [this case](https://chanzuckerberg.github.io/edu-design-system/?path=/story/components-button--just-disabled-prop))

Instead of just a visual warning, this now will include some console warnings (or errors)